src/PolyhedralGeometry/SubdivisionOfPoints/constructors.jl

```diff
@@ -12,11 +12,11 @@ struct SubdivisionOfPoints
 end
 
 @doc Markdown.doc"""
-    SubdivisionOfPoints(Points, Cells)
+    SubdivisionOfPoints(Points::Union{Oscar.MatElem,AbstractMatrix}, Incidence::IncidenceMatrix)
 
 # Arguments
 - `Points::Matrix`: Points generating the cells of the subdivision; encoded row-wise as representative vectors.
-- `Cells::IncidenceMatrix`: An incidence matrix; there is a 1 at position (i,j) if cell i contains point j, and 0 otherwise.
+- `Incidence::IncidenceMatrix`: An incidence matrix; there is a 1 at position (i,j) if cell i contains point j, and 0 otherwise.
```

Comment:
The idea is to make the distinction between the second argument as an `IncidenceMatrix` versus a vector of vectors of integers indexing the maximal cells: i.e. you can call `SubidivisionOfPoints(V,[[1,2,3],[2,3,4]])` where the argument [[1,2,3],[2,3,4]] is more aptly called Cells than Incidence. This is, for example, the format of the elements of the output of `all_triangulations(P)`.

Reply:
On second glance, I understand what you mean better. Incidence tells us nothing.
How do you feel about `Incidence` -> `IncidenceOfCells`?
I am wanting to reserve the term `Cells` for input like `[[1,2,3],[2,3,4]]`. Perhaps there is a better name for this instead?

Reply:
`IncidenceOfCells` or `CellIncidences` is fine.

Reply:
Or maybe `CellIndices`? I think we started to call these things `*Indices` at some point. Maybe @alexej-jordan can say what the right name would be.

Reply:
It does represent exactly the same data, namely the cells, it can be constructed from the same input and it even looks almost identical when printing:
```julia
julia> [[1,2,3],[2,3,4]]
2-element Vector{Vector{Int64}}:
 [1, 2, 3]
 [2, 3, 4]

julia> IncidenceMatrix([[1,2,3],[2,3,4]])
2×4 IncidenceMatrix
[1, 2, 3]
[2, 3, 4]
```

- `IncidenceOfCells` to me sounds like neighbor-relations between cells
- `CellIndices` would be (in my interpretation) for non-consecutive numbers assigned to the cells

I would really prefer just `cells`, especially since the type is also specified and explained.
If you prefer on a different name, I think `cell_point_incidences` would be more precise. Also we usually use `CamelCase` for types and `snake_case` for everything else.

Reply:
Sorry, the `CellIndices` were my suggestion, but I deleted the comment. Also the capitalization is my fault.